.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```

Comment:
I'm tempted to leave this out for now - or add an "other" option.

Reply:
I'll remove it, and we can add the option back later (or add an "Other" option) if we think it's necessary. Is that ok?